controls/radlayoutcontrol/features/toolbox/overview.md

```diff
@@ -0,0 +1,43 @@
+---
+title: ToolBox Overview
+page_title: ToolBox Overview
+description: ToolBox Overview
+slug: radlayoutcontrol-features-toolbox-overview
+tags: layout, toolbox, toolboxview, layoutcontroltoolbox, layoutcontroltoolboxview
+published: True
+position: 0
+---
+
+# ToolBox Overview
+
+The layout control toolbox allows you to add items in the layout control at runtime by drag&drop and also gives you access to the history of the deleted items.
+
+>important The drag/drop operation from the toolbox to the layout control will be allowed only if the layout control is in [edit mode]({%slug radlayoutcontrol-edit-the-layout%}).
+
+## Structure
+
+#### __Figure 1: LayoutControlToolBoxView__
+![](images/radlayoutcontrol-features-toolbox-overview-01.png)
+
+The toolbox has a few sections containing information about the available items (__New Items__), the deleted items (__Deleted Items__) and also the structure of the elements in the layout.
+
+The __New Items__ section of the Items tab displays a list of elements that can be dropped into the layout control.
+
+The __Deleted Items__ of the Items section displays all the elements that have been deleted, and also allows you to drag them back into the layout control. The deleted items feature is disabled by default. To enable it you can use the __TrackDeletedItems__ property of __LayoutControlToolBoxView__.
+
+The __Structure__ tab displays a hierarchical structure of the layout, allowing easy navigation between the parent and child elements. Selecting an item from the tree will select it also in the layout control.
+
+## Setting up the toolbox
+
+The layout control toolbox is represented by the __LayoutControlToolBoxView__ control which you can display in couple ways.
+
+* [Using the LayoutControlToolBox control]({%slug radlayoutcontrol-features-toolbox-layoutcontroltoolbox%}): You can define the control in the __AdditionalCanvasItems__ collection of __RadLayoutControl__. This will embed the toolbox into the layout control.
```

Comment:
Maybe we can say that "This will embed a toggle button into the layout control allowing you to show and hide the LayoutControlToolBoxView."